cmd/influx/main_test.go

```diff
@@ -75,3 +82,161 @@ func TestParseCommand_Use(t *testing.T) {
 		}
 	}
 }
+
+func TestQuery_NoAuth(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping TestQuery_NoAuth")
+	}
+
+	// Create root path to server.
+	// Remove it to clean up past failed panics
+	// Defer it to clean up for successful tests
+	path := tempfile()
+	os.Remove(path)
```

Comment:
`tempfile()` already removes the path, and cleaning up past runs is not possible, since the path changes each time.